### PR TITLE
Arbitrary section

### PR DESCRIPTION
**Pull request by fedefernandez**
This PR adds the `Arbitrary` section for scalacheck exercises.

Please @franciscodr could you review? Thanks

@dialelo @raulraja thoughts?

### CODE REVIEW

File: src/main/scala/scalachecklib/ArbitrarySection.scala [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    */
+  def implicitArbitraryChar(res0: Seq[Char]) = {
+
+    import org.scalacheck.Arbitrary
+    import org.scalacheck.Gen
+    import org.scalacheck.Prop.forAll
+
+    implicit lazy val myCharArbitrary = Arbitrary(Gen.oneOf('A', 'E', 'I', 'O', 'U'))
+
+    val validChars: Seq[Char] = res0
```
Comment (franciscodr) [inline src/main/scala/scalachecklib/ArbitrarySection.scala]: What's the purpose of `validChars`? To give a more specific meaning to `res0` val?

**Comment by franciscodr** [comment]
Minor comment. LGTM

File: src/main/scala/scalachecklib/ArbitrarySection.scala (hunk 1 above)
Comment (fedefernandez) [inline src/main/scala/scalachecklib/ArbitrarySection.scala]: Yes, just to provide more information to the user

Comment (franciscodr) [inline src/main/scala/scalachecklib/ArbitrarySection.scala]: 👍

**Comment by dialelo** [comment]
LGTM!

**Pull request merged**